code/datums/mob_stats.dm

```diff
@@ -236,11 +246,17 @@
 
 //Unused but might be good for later additions
 /datum/stat/proc/setValue_withcap(value)
-	if(value > STAT_VALUE_MAXIMUM)
-		src.value = STAT_VALUE_MAXIMUM
+	if(value > stat_cap)
+		src.value = stat_cap
 	else
 		src.value = value
 
+/datum/stat/proc/change_stat_cap(amount)
```

Comment:
```suggestion
/datum/stat/proc/add_stat_cap(amount)
```
A proc named "change stat cap" is unclear whether it adds the value or sets a new one.